src/webgl/material.js

```diff
@@ -127,11 +130,25 @@ p5.prototype.loadShader = function(
  * // the 'varying's are shared between both vertex & fragment shaders
  * let varying = 'precision highp float; varying vec2 vPos;';
  *
+ * Shaders can alter the positioning of shapes drawn with them.
+ * To ensure consistency in rendering, it's recommended to use the vertex shader, as in the example below.
+ *
  * // the vertex shader is called for each vertex
- * let vs =
- *   varying +
- *   'attribute vec3 aPosition;' +
- *   'void main() { vPos = (gl_Position = vec4(aPosition,1.0)).xy; }';
+ *let vs = `
```

Comment:
To add consistent indentation:
```suggestion
 * let vs = `
```